sdk/ai/azure-ai-projects/samples/agents/sample_agents_with_file_search_attachment.py

```diff
@@ -56,7 +60,10 @@
 
     # Create a message with the file search attachment
     # Notice that vector store is created temporarily when using attachments with a default expiration policy of seven days.
-    attachment = MessageAttachment(file_id=file.id, tools=FileSearchTool().definitions)
+    attachment = MessageAttachment(
+        file_id=file.id,
+        tools=cast(List[MessageAttachmentToolDefinition], FileSearchTool().definitions)
```

Comment:
tools=cast(List[MessageAttachmentToolDefinition], FileSearchTool().definitions)

Was this Pylance error that spotted this? I wonder if we should create task to make this cleaner without cast

Reply:
Removed this code.